packages/lesswrong/components/editor/EditorFormComponent.jsx

```diff
@@ -446,7 +449,10 @@ class EditorFormComponent extends Component {
 
   handleUpdateVersion = async (document) => {
     if (!document) return
-    const editorType = document.contents?.originalContents?.type
+    if (!document.contents?.originalContents) {
```

Comment:
This error appears in console but doesn't isn't very visible to the user. (The user experiences "continuing to edit the latest version of the document". If they switch versions, nothing happens, but they can keep editing as if it's the latest one)

Reply:
(would prefer to figure out how to propagate the error in a way that makes it more clear to the user something is broken)